packages/test-vectors/src/generators/LEIevidenceDocument.js

```diff
@@ -0,0 +1,93 @@
+const faker = require('faker');
+
+const getLEIevidenceDocument = () => {
+  const example = {
+    '@context': ['https://w3id.org/traceability/v1'],
+    type: 'LEIevidenceDocument',
+    "data": {
+      "type": faker.lorem.word(),
```

Comment:
can we make this a hypothetical product instead of a random word?

Reply:
updated, all data aligned with data in real schemas